src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -213,9 +213,9 @@ def default_api_version(self):
         ResourceType.MGMT_ARO: '2020-04-30',
         ResourceType.MGMT_DATABOXEDGE: '2019-08-01',
         ResourceType.MGMT_CUSTOMLOCATION: '2021-03-15-preview',
-        ResourceType.MGMT_CONTAINERSERVICE: SDKProfile('2021-03-01', {
+        ResourceType.MGMT_CONTAINERSERVICE: SDKProfile('2021-05-01', {
```

Comment:
Could you please tell more about the purpose of the version change? (one is upgrade, the other is downgrade) @FumingZhang

Reply:
The purpose of this PR is to upgrade the API version used when the SDK (azure-mgmt-containerservice) sends the actual request when the AKS command is executed,  so we change the version from '2021-03-01' to '2021-05-01' in the “latest” profile. Besides, we will take the new features added by this API version to provide new commands and options.

The downgrade is a fix for the mistake introduced by a previous [PR#18118](https://github.com/Azure/azure-cli/pull/18118) and explained in the following comment.